src/CqlBuilderPanel/functionsSection/Functions.scss

```diff
@@ -67,6 +67,10 @@
     > .cancel-button {
       margin-top: 1rem;
       margin-right: 32px;
+      display: none;
```

Comment:
Although we've confirmed that this display none rule isn't hitting, we should probably not include it in the pr.

Reply:
whoops forgot to remove that. thanks!